build/stf-run-ci/tasks/setup_base.yml

```diff
@@ -13,10 +13,23 @@
           name: certified-operators
         - disabled: false
           name: redhat-operators
-        - disabled: "{{ false if __service_telemetry_observability_strategy in ['use_community', 'use_hybrid'] else true }}"
+        - disabled: false
```

Comment:
I moved this conditional so that ObO could be installed. There are priority rules on the CatalogSources which will dictate order of installation preference with dependency resolution. With local builds (non-index based) we specify the source Catalog.

Ideally in the future we'll be able to update the ObO installation source to redhat-operators vs community-operators.